public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx

```diff
@@ -153,4 +141,29 @@ function getCollapsedInfo(
   return items;
 }
 
+export async function makeStatsRequest(
```

Comment:
Also 1 more thing, I'd change `makeStatsRequest` so that it would return undefined/Query stats. And then set it outside of this function, in each component. Because `makeStatsRequest` is now also setting stats, but also sometimes returning undefined...